Gemfile

```diff
@@ -109,7 +109,7 @@ gem 'statsd-instrument', '~> 2.6.0' # versions beyond 2.6 deprecate config and c
 gem 'swagger-blocks'
 gem 'typhoeus'
 gem 'utf8-cleaner'
-gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', branch: 'master'
+gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', branch: '10-10cg-new-version'
```

Comment:
What's the reason this change isn't merged into `master` on `vets-json-schema` ?

Reply:
I reference the `10-10cg-new-version` branch to ensure that the new schema changes will work with this version of vets-api code. I'll be changing this vets_json_schema reference back to master before merging (after the [schema changes are merged to master](https://github.com/department-of-veterans-affairs/vets-json-schema/pull/537)).

Reply:
Ok, awesome. That PR is approved, so once merged this line will change to reference the `master` branch. Thanks for clarifying!